master/buildbot/test/__init__.py

```diff
@@ -38,6 +41,28 @@
     raise ImportError("\nBuildbot tests require mock version 0.8.0 or "
                       "higher; try 'pip install -U mock'")
 
+
+# This is where we load deprecated module-level APIs to ignore warning produced by importing them.
+# After the deprecated API has been removed, leave at least one instance of the import in a
+# commented state as reference.
+
+
+with assertProducesWarnings(DeprecatedApiWarning,
```

Comment:
I would have put that in a separate test suite, to make sure that all other parts of buildbot are not importing status.
Here we will also be ignoring the import of status in other modules

Reply:
In this case it's impossible, because many parts of the master import status everywhere. This can be solved only by removing it :)